k8s/demo/common/money-claims/cmc-ccd.yaml

```diff
@@ -112,6 +112,8 @@ spec:
         environment:
           S2S_NAMES_WHITELIST: ccd_gateway,cmc,cmc_claim_store,ccd_data,dg_docassembly_api
     ccd-user-profile-importer:
+      java:
```

Comment:
There shouldn't be java in here. This chart is not based on java chart.

Reply:
removed.